tests/unit/BlockStorageMultiSig.d

```diff
@@ -87,7 +87,7 @@ private void main ()
 
     /// Update each block adding an updated block multisig and set validator 0
     /// as also signed
-    iota(1, BlockCount).each!(h => {
+    iota(1, BlockCount).each!((h) {
```

Comment:
I have still seen cases where compile fails and it doesn't give the line number of the actual line that causes the problem. Only solution I think is to call inner function in lambda.